### PR TITLE
feat(web): assets expand animation

### PR DESCRIPTION
**Pull request by JasBogans**

### CODE REVIEW

**Comment by alextran1502** [comment]
Hello, can you please add some screen recording for the feature? Thank you

**Comment by JasBogans** [comment]
Yeah no problem

https://github.com/immich-app/immich/assets/12576070/65477251-ab2c-4871-abb5-bb2ac4cd8168

File: web/src/lib/components/elements/spinner.svelte
[the remark this answers is not in the document]
Comment (JasBogans) [reply web/src/lib/components/elements/spinner.svelte:1]: Sorry my mistake, I was looking for the [web/src/lib/components/elements/spinner.svelte](https://github.com/immich-app/immich/pull/4181/files/6e86952b91386c455bab7e5237c27524a41b7ba8#diff-fc81309a009aae4ad2e2d168214abee37b20981925d6d0c3b076c0baab8f5925) but I didn't find it, I'll try to use this and then make a new commit if they don't differe.
Btw how can I create a draft with a to do list from this PR?

[the remark this answers is not in the document]
Comment (jrasm91) [reply web/src/lib/components/elements/spinner.svelte:1]: Like this:
```markdown
- [ ] WIP
- [x] Done!
``` 
- [ ] WIP
- [x] Done!

[the remark this answers is not in the document]
Comment (jrasm91) [reply web/src/lib/components/elements/spinner.svelte:1]: If it is still work in progress, you can convert it to a draft on the web in the top right corner
image

**Comment by ayykamp** [comment]
would probably be best to add a shrink animation when navigating back as well as well

**Comment by alextran1502** [comment]
Stale PR

**Pull request closed without merging**